### PR TITLE
Expense group actions

### PR DESCRIPTION
**Pull request by mohammadamin-alizadeh**
The items that I did in this PR:

- Implement `Local Storage` as DB
- Add **CRUD** feature to expense group

### CODE REVIEW

File: src/components/DeleteAlert.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const DeleteAlert = () => {
+  return (
+    <div
+      className="p-4 mb-4 text-sm text-red-800 rounded-lg bg-red-50 dark:bg-gray-800 dark:text-red-400"
+      role="alert"
+    >
+      <span className="font-medium">Removed!</span>
+      You removed a expense group
```
Comment (parykhan-jameel) [inline src/components/DeleteAlert.tsx]: here you can only use one warning message, this is how it shows when a user removes a group:
Screenshot 2024-09-24 220544
I feel like it's doesn't give it a nice view.

File: src/components/SuccessAlert.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const SuccessAlert = () => {
+  return (
+    <div
+      className="p-4 mb-4 text-sm text-green-800 rounded-lg bg-green-50 dark:bg-gray-800 dark:text-green-400"
+      role="alert"
+    >
+      <span className="font-medium">Awesome!</span>
+      You Successfully create an expense group
```
Comment (parykhan-jameel) [inline src/components/SuccessAlert.tsx]: same thing here, or you can just give some space between them

**Review by parykhan-jameel — CHANGES_REQUESTED** [review]
only those two tiny comments I've added, otherwise is all perfect.

**Review by MarwanBz — APPROVED** [review]
looks amazing bro!
thanks

File: src/components/SuccessAlert.tsx (hunk 2 above)
Comment (mohammadamin-alizadeh) [reply src/components/SuccessAlert.tsx]: @parykhan-jameel I changed it to one sentence.

**Review by parykhan-jameel — APPROVED** [review]
amazing work!

**Review by MarwanBz — APPROVED** [review]

**Pull request merged**